kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/bind.kt

```diff
@@ -1,48 +1,116 @@
 package io.kotest.property.arbitrary
 
-fun <A, T> Arb.Companion.bind(arbA: Arb<A>, createFn: (A) -> T): Arb<T> = arb {
-   createFn(arbA.single(it))
-}
+import io.kotest.property.RandomSource
+import io.kotest.property.Sample
+
+fun <A, T> Arb.Companion.bind(arbA: Arb<A>, createFn: (A) -> T): Arb<T> = arbA.map(createFn)
```

Comment:
Note the change here. Is this bind worth keeping since it is equivalent to map?

Reply:
Probably not. Lets remove it.